test/SwiftSyntax/Inputs/TokenKindList.txt.gyb

```diff
@@ -0,0 +1,6 @@
+%{
+  from gyb_syntax_support import *
+}%
+% for token in SYNTAX_TOKENS:
+${token.kind}
+% end
```

Comment:
Add an action `print-token-kind` in swift-swiftsyntax-test for this output.

Reply:
I actually think that this way is better. This is not really swift-specific since we're just checking the list of declarations in the python declaration of `gyb_syntax_support`.

If we were to implement this in `swift-swiftsyntax-test` we would need to:
a) Make `swift-swiftsyntax-test` be a `gyb` tool (I don't like this at all)
b) Need to check that all the tokenKinds get generated in the `TokenKind` enum, but for that we'd need to have a list of all kinds define in `TokenKind` and we cannot use the autogenerated `allCases` property using `CaseIterable` since `TokenKind` has associated values.

And after all this file is not a real tool that needs to be compiled and is a standalone binary, but is just an input to `gyb`.

Reply:
@nkcsgexi and I discussed this in person and decided that it's probably the easiest and cleanest way to test this. In the future we might want to consider generation `TokenKinds.def` from `gyb` which would make the entire test obsolete.